cmd/controller/controller.go

```diff
@@ -48,11 +49,11 @@ func init() {
 func main() {
 	cfg, err := clientcmd.BuildConfigFromFlags(masterURL, configPath)
 	if err != nil {
-		glog.Fatalf("Error getting kube config: %v\n", err)
+		glog.Fatalf("%+v", errors.Wrap(err, "Unable to get kube config"))
```

Comment:
It looks like `[glog.Fatal](https://github.com/golang/glog/blob/master/glog.go#L1144)` already provides a trace